File: java/1009-salario_com_bonus.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
  public static void main(String[] args) throws IOException {
    String nome;
    double salario, vendas, total;

    Scanner scanner = new Scanner(System.in);
    nome = scanner.nextLine();
    salario = scanner.nextDouble();
    vendas = scanner.nextDouble();
    scanner.close();

    total = salario + (vendas * .15);
    System.out.printf("TOTAL = R$ %.2f\n", total);
  }
}

File: kotlin/1009-salario_com_bonus.kt
import java.util.*

fun main(args: Array<String>) {
  val NOME = readLine()
  val SALARIO = readLine()!!.toDouble()
  val VENDAS = readLine()!!.toDouble()
  val TOTAL = SALARIO+VENDAS*.15
  println("TOTAL = R$ " + "%.2f".format(TOTAL))
}
